### PR TITLE
bunch of project updates

### PR DESCRIPTION
**Pull request by bishabosha**

### CODE REVIEW

File: _projects/10-scala-native.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ contributors: [martinduhem, guillaume, ergys]
 status: Active
 type: project
 active: true
-priority: 1
+priority: 2
```
Comment (sjrd) [inline _projects/10-scala-native.md:11]: What does this affect, actually?

**Review by sjrd — APPROVED** [review]

File: _data/team-members.yml [hunk 2]
```diff
@@ -194,10 +194,12 @@ members:
     twitter: adrienpi2
     photo: /resources/img/adrien.png
     currentlyWorkingOn: true
-    contact_for: [scaladex]
+    contact_for: [scala3MigrationGuide]
```
Comment (julienrf) [inline _data/team-members.yml:197]:
```suggestion
    contact_for: [scaladex,scala3MigrationGuide]
```

If this is fine to @adpi2

File: _data/team-members.yml [hunk 3]
```diff
@@ -194,10 +194,12 @@ members:
...
     projects:
+        - title: scala3MigrationGuide
+          percentage: 90
         - title: scaladex
-          percentage: 100
+          percentage: 10
```
Comment (julienrf) [inline _data/team-members.yml:202]: Should we also add BSP implementation in sbt?

File: _projects/10-scala-native.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ contributors: [martinduhem, guillaume, ergys]
 status: Active
```
Comment (julienrf) [inline _projects/10-scala-native.md:8]: I just noticed that we should add @errikos to the contributors list

Comment (julienrf) [reply _projects/10-scala-native.md:8]: (and after #140 is done we should add Wojciech)

**Review by julienrf — COMMENTED** [review]
Thank you for updating the pages, @bishabosha :)

I’m fine with the changes you proposed but I’ve left a few comments, if this is OK for you

File: _projects/10-scala-native.md (hunk 1 above)
Comment (bishabosha) [reply _projects/10-scala-native.md:11]: should just make it higher in the list than SIP

File: _projects/10-scala-native.md (hunk 4 above)
Comment (bishabosha) [reply _projects/10-scala-native.md:8]: looks like Ergys is already there

File: _data/team-members.yml (hunk 3 above)
Comment (bishabosha) [reply _data/team-members.yml:204]: done

File: _data/team-members.yml (hunk 2 above)
Comment (bishabosha) [reply _data/team-members.yml]: done

**Comment by bishabosha** [comment]
I addressed the changes here

File: _projects/32-scala-2-to-3-migrator.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+label: migrat3
+name: lorem ipsum
```
[the remark this answers is not in the document]
Comment (bishabosha) [reply _projects/32-scala-2-to-3-migrator.md]: whoops, thank you

**Review by julienrf — APPROVED** [review]

**Pull request merged**